backend/migrations/default/1668093456409_run_sql_migration/up.sql

```diff
@@ -0,0 +1,17 @@
+create or replace function core.reorder_indexes()
```

Comment:
This would result in incrementing the indices of every section with index >= NEW.index, even if that section is in another document.
- [ ] Fix reoder_indexes()